Makefile

```diff
@@ -27,8 +27,14 @@ clean-pyc:
 	find . -name '*.pyo' -exec rm -f {} +
 	find . -name '*~' -exec rm -f {} +
 
+format:
+	isort mozilla_schema_generator tests ./*.py
+	black mozilla_schema_generator tests ./*.py
```

Comment:
@wlach instead of lint-fix I've added `make format`  to format the code. As for autoflake, I will add that in the next commit.